Datagrid/SimplePager.php

```diff
@@ -13,6 +13,9 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 
+/**
+ * @deprecated Will be removed in version 2.0. Use Sonata\AdminBundle\Datagrid\SimplePager instead.
+ */
```

Comment:
So .. i got Deprecation-Warning in constructor (?), The `NEXT_MAJOR` flag, and what to do with the tests?